.travis.yml

```diff
@@ -10,7 +10,9 @@ services:
   - docker
 
 before_install:
+  - ./tools/travis/setupscan.sh
```

Comment:
All the other repos use the ./tools/travis conventions. I called Daniel Lavine and asked him if there was some reason to use ./.travis, but he could not think of any and my searches did not yield anything meaningful (although I did see a few repos/projects used this as the location). Please let me know if there is special meaning to using ./.travis as if there is a compelling reason, we may want to change all the other repos. as well. In any event, it would be desirable to have all repos. use the same directory conventions where possible IMO; it also helps with setting up "path exclusions" for uniform code scanning as well.